make/tests

```diff
@@ -21,6 +23,17 @@ find_test() {
 TEST_FILE="$(find_test)"
 shift
 
+if [ "${POD_NAME}" == "acceptance-tests" ]; then
+    UAA_COUNT=$(kubectl get sts --namespace "${UAA_NAMESPACE}" uaa -o jsonpath='{.status.replicas}')
+    if [ ${UAA_COUNT} -gt 1 ]; then
+        echo -e "\033[36mThe SSO lifecycle tests are known to fail with UAA in HA mode.\033[0m"
+        echo -e "\033[36mMake you have disabled the tests with e.g. \033[32menv.CATS_SUITES=-sso\033[0m"
```

Comment:
Prefer `printf` over `echo -e`, because `dash` is annoying and prints `-e` directly. Doesn't matter here, though, since we're explicitly running via `bash`.